README.md

```diff
@@ -1,23 +1,57 @@
-# Next.js + Jest
-
-This example shows how to configure Jest to work with Next.js.
-
-This includes Next.js' built-in support for Global CSS, CSS Modules, and TypeScript!
-
-## How to Use
-
-Quickly get started using [Create Next App](https://github.com/vercel/next.js/tree/canary/packages/create-next-app#readme)!
-
-In your terminal, run the following command:
-
-```bash
-npx create-next-app --example with-jest with-jest-app
-# or
-yarn create next-app --example with-jest with-jest-app
-```
-
-## Run Jest Tests
-
-```bash
-npm test
-```
+# Sparkbox Apprentice Page
+
+## Contributing 
+
+##### Git:
+
+- When you want to make a contribution please follow the [Sparkbox git process](https://github.com/sparkbox/standard/tree/main/code-style/git) for naming branches, writing commit messages, and rebasing before submitting PRs.
+
+##### Updating Apprentices Data:
+
+  ###### Images: 
+
+  - All image files should be added to ```/public/apprentices``` with the file name being ```[ApprenticeName].png```
+
+  ###### Apprentice Data:
+
+  - To add/change the current apprentices or previous apprentices data navigate to ```/data``` folder and find the ```apprentices.json``` file
+
+  - Add to and/or adjust the data, see below how the data for each group is structured:
+    - *Current Apprentices*:
+    ```js
+    "currentApprentices": {
+      "version": "[Version Number]",
+      "apprentices": [
+        ...,
+        {
+          "name": "[Apprentice Name]",
+          "image": "/apprentices/[Name].png",
+          /* Links are optional and can be any link the apprentice provides as long as it follows 
+          the structure of the objects below: */
+          "links": [
+            { "href": "https://www.personalportfolio.com", "text": "Personal" },
+            { "href": "https://www.linkedin.com/", "text": "Linkedin" },
+            { "href": "https://www.github.com/", "text": "Github" },
+          ],
+        },
+      ],
+    },
+     ```
+    - *Previous Apprentices*:
+    ```js 
+    "previousApprentices": [
+      ...,
+      {
+        "version": "[Version Number]",
+        "apprentices": [
+          ...,
+          {
+            "name": "[Apprentice Name]",
+            /* Status is optional and states the apprentice employment status with Sparkbox
+            and will be one of the following values: */
+            "status": [ "current" || "previous" || null ]
```

Comment:
`status` is an optional parameter, correct? I believe the component will behave the same whether the value is null or if the property is completely undefined. We may want to guide toward omitting `status` entirely when unnecessary, rather than setting `null`.